README.md

```diff
@@ -1,11 +1,81 @@
 # Orion
 Orion is a PegaSys component for doing private transactions.
 
-
 [![Gitter chat](https://badges.gitter.im/ConsenSys/orion.png)](https://gitter.im/ConsenSys/orion)
 [![CircleCI](https://circleci.com/gh/ConsenSys/orion.svg?style=shield&circle-token=5f92fd966a971e60e57f53f2257fe5dda0fcf52c)](https://circleci.com/gh/ConsenSys/orion)
 [![Download](https://api.bintray.com/packages/consensys/binaries/orion/images/download.svg)](https://bintray.com/consensys/binaries/orion/_latestVersion)
 
+## Quick Start Guide
+
+### Download binary
+Download the latest binary from [Bintray](https://bintray.com/consensys/binaries/orion/_latestVersion) and extract the folder:
+```
+tar -xvzf orion*.tar.gz
+cd orion-<version>/bin
+```
+
+### Generate keys
+When generating keys, you must specify a name:
+```
+$ ./orion -g orion
+```
+This will generate `orion.key` and `orion.pub` files.
+
+### Create config file
+Create a file `orion.conf` and add the following properties:
+```
+nodeurl = "http://127.0.0.1:8080/"
+nodeport = 8080
+clienturl = "http://127.0.0.1:8888/"
+clientport = 8888
+publickeys = ["orion.pub"]
+privatekeys = ["orion.key"]
+tls = "off"
+```
+
+### Start Orion
+Run Orion with the created config file:
+```
+$ ./orion orion.conf
+```
+
+### Check if Orion is up and running
+The Upcheck method can be used to test if Orion is up and running:
+```
+$ curl http://localhost:8888/upcheck
+$ I'm up!
+```
+### Send a payload
+With only one node running, you can only send a payload to yourself:
+```
+$ curl -X POST \
+  http://localhost:8888/send \
+  -H 'Content-Type: application/json' \
+  -d '{ 
+        "payload": "SGVsbG8sIFdvcmxkIQ==",
+        "from": "4xanJzyaDPcBVMUSwl/tLp+DbXzd3jF9MKk1yJuyewE=",
+        "to": ["4xanJzyaDPcBVMUSwl/tLp+DbXzd3jF9MKk1yJuyewE="]
+      }'
+
+$ {"key":"LcF7I+UnR2XBdSxZesiYE/lTtxVfFeY4EvL9fDXb0Uo="}
+```
+`from` and `to` values are the public key generated in a previous step (`orion.pub`).
+
+### Receive a payload
+Using the key from the Send method, we can receive the payload:
+```
+$ curl -X POST \
+  http://localhost:8888/receive \
+  -H 'Content-Type: application/json' \
+  -d '{
+        "key": "LcF7I+UnR2XBdSxZesiYE/lTtxVfFeY4EvL9fDXb0Uo=",
```

Comment:
These value are dependent on output from the earlier key generation step, so will be different when the user generates their own key